libcnb-data/src/newtypes.rs

```diff
@@ -3,13 +3,14 @@
 /// Automatically implements the following traits for the newtype:
 /// - [`Clone`]
 /// - [`Debug`]
-/// - [`Display`]
+/// - [`Display`](std::fmt::Display)
 /// - [`Eq`]
 /// - [`PartialEq`]
 /// - [`serde::Deserialize`]
 /// - [`serde::Serialize`]
-/// - [`FromStr`]
-/// - [`Borrow<String>`]
+/// - [`FromStr`](std::str::FromStr)
+/// - [`Borrow<String>`](std::borrow::Borrow<String>)
+/// - [`Borrow<str>`](std::borrow::Borrow<String>)
```

Comment:
Sneaky. ;-)

Reply:
This should also actually be `std::borrow::Borrow<str>` not `std::borrow::Borrow<String>` - I'll update now.